README.md

```diff
@@ -18,6 +18,12 @@ For all other storage systems, please check with your storage vendor.  If your s
 durable snapshot storage you may want to consider 
 [Velero's Restic Integration](https://velero.io/docs/latest/restic)
 
+## Verified environment
+Velero CSI plugin v0.3 is tested with Velero v1.9 on the following environments:
+* EKS: ebs.csi.aws.com driver is tested with helm chart version helm-chart-aws-ebs-csi-driver-2.6.8 and external-snapshotter version is v5.0.1. v6.0.1 external-snapshotter is also tested with EBS CSI driver helm chart version helm-chart-aws-ebs-csi-driver-2.6.11.
+* AKS: disk.csi.azure.com driver is tested. CSI related controllers are installed in control-plan, so cannot find the exactly version. Tested AKS versions are 1.23.3 and v1.22.6.
```

Comment:
Sorry just saw another trivial issue:
It should be `on control plane`, not `in control-plan`

Reply:
Modified.